boat-scaffold/src/main/templates/boat-angular/apiMocks.mustache

```diff
@@ -1,76 +1,32 @@
-import { createMocks } from '@backbase/foundation-ang/data-http';
-import { Provider } from '@angular/core';
-
-{{#operations}}
-    {{#operation}}
-/**
-* Mocks provider for {{{basePathWithoutHost}}}{{pattern}} URL pattern
-*/
-export const {{classname}}{{#lambda.titlecase}}{{nickname}}{{/lambda.titlecase}}MocksProvider = /*#__PURE__*/ createMocks([{
-        urlPattern: "{{{basePathWithoutHost}}}{{pattern}}",
-        method: "{{#lambda.uppercase}}{{httpMethod}}{{/lambda.uppercase}}",
-        responses: [
-        {{#responses}}
-            {{#examples}}
-                {{#isJson}}
-                {
-                    status: {{{code}}},
-                    body: {{{prettyPrintValue}}}
-                },
-                {{/isJson}}
-                {{^isJson}}
-                    {
-                    status: {{{code}}},
-                    body: "{{#escapeJavascript}}{{{prettyPrintValue}}}{{/escapeJavascript}}"
-                    },
-                {{/isJson}}
-            {{/examples}}
-            {{#hasEmptyBody}}
-                {
-                status: {{{code}}},
-                body: ""
-                },
-            {{/hasEmptyBody}}
-        {{/responses}}
-    ]
-}]);
-    {{/operation}}
-{{/operations}}
-
-export const {{classname}}MocksProvider = /*#__PURE__*/ createMocks(
-    [{{#operations}}
+module.exports = [{{#operations}}
```

Comment:
Should this be `export default` not `module.exports = `?

Reply:
whatever I tried, I ended up with error like `Unexpected token 'export'`. From the way it compiles, `module.exports` suits best. `.js` and `.ts` doesn't matter, imo. 
Why should we move them? the structure stays the same, mocks are where they were before, only format is different

Reply:
> module.exports suits best. .js and .ts doesn't matter, imo

This is currently generated as a `.ts` file, not as a `.js` one.  If the intention is to create .js files, we'd need to change that in the generator [here](https://github.com/Backbase/backbase-openapi-tools/blob/b1a981f2235d2df2ce73579123c3924b851f0f88/boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java#L226).

> Why should we move them?

Just thought it would be nice to have them collected in one place, away from everything else.  Especially since they're not designed to be used as actual runtime code (unlike the old mocks).

Reply:
Yes, I understand where to change it. What I meant is that a format could be either of the options. We don't have any types there, so it could be `js`; at the same time for uniformity we can keep it as `ts`.

@cheshirsky what do you think about mocks placement? For me both options make sense.

Reply:
> for uniformity we can keep it as `ts`.

The problem is `module.exports = ...` isn't valid typescript.  You'll get a compile error with this as-is.

You can write typescript with exports that _compile to_ the commonjs format, though.  Either by adding `declare let module: any;` at the start, or by chaging the "module" setting in tsconfig to "CommonJS" and using the `export = ...` syntax.

Reply:
ok, I think it would be better to have mocks as `js`, as they don't have anything that requires or uses typescript.